internal/pkg/print/markdown/table/table.go

```diff
@@ -64,10 +64,10 @@ const (
 		{{ if not .Module.Outputs }}
 			No output.
 		{{ else }}
-			| Name | Description |
-			|------|-------------|
+			| Name | Description | {{if $.Settings.OutputValues}}Value |{{end}}
```

Comment:
Please move the leading space from before {{if &.Settings....` to inside the if.

And also put surrounding spaces inside `{{ ... }}`, e.g. `{{ end }}` instead of `{{end}}`.

Reply:
- [X] Fix Spacing